ts/screens/authentication/LandingScreen.tsx

```diff
@@ -32,6 +32,9 @@ type OwnProps = {
 };
 
 type Props = ReduxProps & OwnProps;
+const isCIEAuthenticationSupported = false; // waiting for sdk cie implementation
```

Comment:
```suggestion
const isCIEAuthenticationSupported = false; // TODO: waiting for sdk cie implementation https://www.pivotaltracker.com/story/show/169730204
```

Reply:
It seems you don't add the link to the pivotal story. Can you check it?

Reply:
Fixed